cli/src/build.ts

```diff
@@ -409,7 +409,7 @@ export class BuildCommand extends Command {
       }
     }
 
-    const targetRootDir = await findUp(cwd)
+    const targetRootDir = process.env.CARGO_TARGET_DIR || await findUp(cwd)
```

Comment:
`||` is desired here to catch the case when the env var is an empty string, should I disable eslint for this line?

Reply:
Ah sorry, I missed this comment. I think to add eslint ignore here is reasonable.

Thanks for your PR